force-app/main/default/layouts/Program_Enrollment__c-HEDA Program Enrollment Layout.layout-meta.xml

```diff
@@ -14,62 +14,88 @@
                 <behavior>Required</behavior>
                 <field>Contact__c</field>
             </layoutItems>
+            <layoutItems>
+                <behavior>Required</behavior>
+                <field>Account__c</field>
+            </layoutItems>
             <layoutItems>
                 <behavior>Edit</behavior>
-                <field>Eligible_to_Enroll__c</field>
+                <field>Start_Date__c</field>
             </layoutItems>
             <layoutItems>
                 <behavior>Edit</behavior>
-                <field>Admission_Date__c</field>
+                <field>Class_Standing__c</field>
             </layoutItems>
+        </layoutColumns>
+        <layoutColumns>
             <layoutItems>
```

Comment:
This is missing a layoutItem for a blank space outlined in the screenshot for implementation, changing the tab order of fields.

Reply:
Fixed